cmd/config.go

```diff
@@ -431,29 +435,31 @@ func (c Config) MustYAML() []byte {
 	return out
 }
 
-func defaultConfig(memo string) []byte {
+func defaultConfig(memo string, prometheusListen string) []byte {
 	return Config{
-		Global: newDefaultGlobalConfig(memo),
+		Global: newDefaultGlobalConfig(memo, prometheusListen),
 		Chains: relayer.Chains{},
 		Paths:  relayer.Paths{},
 	}.MustYAML()
 }
 
 // GlobalConfig describes any global relayer settings
 type GlobalConfig struct {
-	APIListenPort  string `yaml:"api-listen-addr" json:"api-listen-addr"`
-	Timeout        string `yaml:"timeout" json:"timeout"`
-	Memo           string `yaml:"memo" json:"memo"`
-	LightCacheSize int    `yaml:"light-cache-size" json:"light-cache-size"`
+	APIListenPort    string `yaml:"api-listen-addr" json:"api-listen-addr"`
+	PrometheusListen string `yaml:"prometheus-listen-addr" json:"prometheus-listen-addr"`
```

Comment:
If I were an operator, I would be surprised I have to worry about 2 ports. To echo Mark, I think http server for both metrics and debug profiling is ideal.

Reply:
consolidated prometheus metrics into the debug server